<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1360</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1360</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reportes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>351</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
  font-weight: bold;
  border: 1px solid #00a2f5;
  border-radius: 4px;
  padding: 4px;
margin-top:8px;
}
QGroupBox::title {
 color:#00a2f5;
  subcontrol-origin: margin;
  subcontrol-position: top left;
  left: 5px;
  padding-left: 3px;
  padding-right: 5px;
  padding-bottom: 16px;
font-weight: bold;
}</string>
   </property>
   <property name="title">
    <string>Infractores Detectados</string>
   </property>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 214, 255);</string>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2021</year>
      <month>2</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="minimumDate">
     <date>
      <year>2021</year>
      <month>2</month>
      <day>1</day>
     </date>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="today_lbl">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Infractores</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>170</y>
     <width>351</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
  font-weight: bold;
  border: 1px solid #00a2f5;
  border-radius: 4px;
  padding: 4px;
margin-top:8px;
}
QGroupBox::title {
 color:#00a2f5;
  subcontrol-origin: margin;
  subcontrol-position: top left;
  left: 5px;
  padding-left: 3px;
  padding-right: 5px;
  padding-bottom: 16px;
font-weight: bold;
}</string>
   </property>
   <property name="title">
    <string>Rostros Detectados</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Con mascarillas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Sin mascarillas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Total</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="with_lbl">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="without_lbl">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="total_lbl">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>351</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
  font-weight: bold;
  border: 1px solid #00a2f5;
  border-radius: 4px;
  padding: 4px;
margin-top:8px;
}
QGroupBox::title {
 color:#00a2f5;
  subcontrol-origin: margin;
  subcontrol-position: top left;
  left: 5px;
  padding-left: 3px;
  padding-right: 5px;
  padding-bottom: 16px;
font-weight: bold;
}</string>
   </property>
   <property name="title">
    <string>Porcentajes de hoy</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>311</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>961</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
  font-weight: bold;
  border: 1px solid #00a2f5;
  border-radius: 4px;
  padding: 4px;
margin-top:8px;
}
QGroupBox::title {
 color:#00a2f5;
  subcontrol-origin: margin;
  subcontrol-position: top left;
  left: 5px;
  padding-left: 3px;
  padding-right: 5px;
  padding-bottom: 16px;
font-weight: bold;
}</string>
   </property>
   <property name="title">
    <string>Hist√≥rico</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>19</y>
      <width>941</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
